src/Clipper.jl

```diff
@@ -306,6 +306,14 @@ function Path(ct::Integer=0)
     @cxx ClipperLib::Path(ct)
 end
 
+function Path(pts::Vector{(Int, Int)}, ct::Integer=0)
```

Comment:
`ct` isn't needed here, because `Path(::Int)` preallocates and this is `push!`ing.